src/codegen/formatted_elz_new/mod.rs

```diff
@@ -0,0 +1,183 @@
+mod tests;
```

Comment:
I say remove old formatter module, not metadata for rust compiler. Crate test attr helps rust compiler know mod tests is testing module. Please at least run cargo test before push any commits.